README.md

```diff
@@ -301,6 +302,31 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+#### grcov with Gitlab
+
+Here is an example `.gitlab-ci.yml` which will build and collect coverage data in Gitlab:
+
+```yaml
+build:
+  script:
+    - cargo test --workspace
```

Comment:
Can you add LLVM_PROFILE_FILE here as well?

Reply:
Done, setting the `variables:` adds an environment variable so it'll be present for all the code execution.